pyproject.toml

```diff
@@ -359,7 +359,7 @@ filterwarnings = [
   "ignore:There is no current event loop:DeprecationWarning:",
   "ignore:(?s).*Pyarrow will become a required dependency of pandas:DeprecationWarning",  # pandas pyarrow (pandas<3.0),
   # TODO: remove once xarray is updated to avoid this warning
-  # https://github.com/pydata/xarray/blame/b1f3fea467f9387ed35c221205a70524f4caa18b/pyproject.toml#L333-L334
+  # https://github.com/pydata/xarray/blame/b1f3fea467ff9387ed35c221205a70524f4caa18b/pyproject.toml#L333-L334
```

Comment:
The link doesn't work; this change is a typo I think -- I spot an extra `f`.

Reply:
Oops that was a stray keypress. Fixed.